### PR TITLE
Variable Fix: NGROK_URL change to SERVER_URL

### PR DESCRIPTION
**Pull request by haseeb-xd**
Issue #696

### CODE REVIEW

**Comment by ieaves** [comment]
Hey @haseeb-xd thanks for the contribution. It looks like `NGROK_URL` is also referenced [here](https://github.com/grai-io/grai-core/blob/df3d407d3dd85cea7fa09bef31dffffa3768d737/grai-server/app/connections/schedules/dbt_cloud.py#L17) and will need to be modified.

File: grai-server/app/the_guide/settings/base.py [hunk 1]
```diff
@@ -326,7 +326,7 @@ def inner(value: str | bool) -> bool:
 GITHUB_APP_ID = config("GITHUB_APP_ID", None)
 GITHUB_PRIVATE_KEY = config("GITHUB_PRIVATE_KEY", None)
 
-NGROK_URL = config("NGROK_URL", f"https://{SERVER_HOST}")
+SERVER_URL = config("NGROK_URL", f"https://{SERVER_HOST}")
```
Comment (ieaves) [inline grai-server/app/the_guide/settings/base.py:329]: Could we also change the environment variable reference from `NGROK_URL` i.e.

```python
SERVER_URL = config("SERVER_URL", f"https://{SERVER_HOST}")
```

**Review by ieaves — CHANGES_REQUESTED** [review]

File: grai-server/app/the_guide/settings/base.py (hunk 1 above)
Comment (haseeb-xd) [reply grai-server/app/the_guide/settings/base.py:329]: yes, I'll fix it.

**Comment by ieaves** [comment]
Failing tests are unrelated to this change. 

LGTM and thanks for the contribution @haseeb-xd!

**Pull request merged**